api/consensus.go

```diff
@@ -4,28 +4,17 @@ import (
 	"net/http"
 
 	"github.com/NebulousLabs/Sia/build"
-	"github.com/NebulousLabs/Sia/crypto"
 	"github.com/NebulousLabs/Sia/types"
 )
 
-// The ConsensusSetStatus struct contains general information about the
-// consensus set, with tags to have idiomatic json encodings.
-type ConsensusSetStatus struct {
+// The ConsensusGET struct contains general information about the consensus
+// set, with tags to support idiomatic json encodings.
+type ConsensusGET struct {
 	Height       types.BlockHeight `json:"height"`
 	CurrentBlock types.BlockID     `json:"currentblock"`
 	Target       types.Target      `json:"target"`
```

Comment:
the latter; if we want to be consistent, we need to add json tags to every field of every struct that gets sent over the API. That's not so bad as long as most of those structs are defined in the api package, though. Perhaps we should enforce that all structs sent over the API **must** be defined in the api package, to avoid that sort of thing (at the expense of some duplicate types).